app/src/main/java/ayds/lisboa/songinfo/home/model/repository/external/spotify/tracks/SpotifyToSongResolver.kt

```diff
@@ -16,18 +19,21 @@ private const val ARTISTS = "artists"
 private const val ALBUM = "album"
 private const val IMAGES = "images"
 private const val RELEASE_DATE = "release_date"
+private const val RELEASE_DATE_PRECISION = "release_date_precision"
 private const val URL = "url"
 private const val EXTERNAL_URL = "external_urls"
 private const val SPOTIFY = "spotify"
 
 internal class JsonToSongResolver : SpotifyToSongResolver {
 
+    private val spotifyReleaseDateMapper: SpotifyReleaseDateMapper = SpotifyHelperInjector.spotifyReleaseDateMapper
```

Comment:
Siempre que se pueda, es mejor inyectar las dependencias por constructor, para evitar la dependencia al injector desde esta clase.
```suggestion
internal class JsonToSongResolver(private val spotifyReleaseDateMapper: SpotifyReleaseDateMapper) : SpotifyToSongResolver {
```